terraform/modules/gro_ingestion_service/lambdas/src/functions/publish-event.function.ts

```diff
@@ -0,0 +1,57 @@
+import {DynamoDBRecord, DynamoDBStreamEvent, Handler} from "aws-lambda"
+import {mapToEventRecord} from "../models/EventRecord"
+import {request} from "https"
+
+const apiUrl = process.env.API_URL
+const options = {
+    hostname: apiUrl,
+    path: "/todos",
```

Comment:
`todos` seems like we should probably change it!

Reply:
Yep